### PR TITLE
[BUILD] cmake: generating lists/groups of sources and headers files moved into separated cmake files

### PR DESCRIPTION
**Pull request by knapikm**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -710,341 +710,26 @@ endif()
...
-# Aruco
-file( GLOB_RECURSE SRC_ARUCO	"src/Aruco/*.cpp" )
-
-# Kinect
-file( GLOB_RECURSE SRC_KINECT	"src/Kinect/*.cpp" )
-
-# SpeechSDK
-file( GLOB_RECURSE SRC_SPEECH	"src/Speech/*.cpp" )
+# add cmake modul that generates a list of source files
+include( cmake/Sources.cmake )
```
Comment (kapecp) [inline CMakeLists.txt]: toto nie je celkom dobre riesenie:

- presunut cmake/Sources.cmake do src/Sources.cmake
- resp. by sa to malo spravit cez add_subdirectory

**Review by kapecp — CHANGES_REQUESTED** [review]
adresar cmake by mal obsahovat iba:

- find skripty
- pomocne build skripty
- NIE buildovanie 3DSoftviz-u

**Comment by KrupsonSVK** [comment]
fixol som to, src/ je ako subdirectory

**Review by KrupsonSVK — APPROVED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -710,341 +710,26 @@ endif()
...
-file( GLOB_RECURSE HEADER_HANDMODULE	"include/Leap/HandModule/*.h" )
-
-# MOUSE3D
-set ( HEADER_MOUSE3D
-		include/Mouse3d/Connector.h
-		include/Mouse3d/LibMouse3d/EventThread.h
-		include/Mouse3d/LibMouse3d/Mouse3dDevices.h
-		)
+# add cmake modul that generates a list of header files
+include (cmake/Headers.cmake)
```
Comment (kapecp) [inline CMakeLists.txt]: prerobit na `include(include/Headers.cmake)`

File: CMakeLists.txt [hunk 3]
```diff
@@ -710,341 +710,26 @@ endif()
...
-
-# Aruco
-file( GLOB_RECURSE SRC_ARUCO	"src/Aruco/*.cpp" )
-
-# Kinect
-file( GLOB_RECURSE SRC_KINECT	"src/Kinect/*.cpp" )
-
-# SpeechSDK
-file( GLOB_RECURSE SRC_SPEECH	"src/Speech/*.cpp" )
+# add cmake module that generates a list of source files
```
Comment (kapecp) [inline CMakeLists.txt]: _CMake module_ sa terminologicky pouziva v inom kontexte, resp. znaci nieco ine
- upravit komentar na: _process src directory - get all source files_ (podobne aj nizsie)
- poprosim pisat _CMake_

**Review by kapecp — CHANGES_REQUESTED** [review]
este par drobnosti opravit

**Comment by KrupsonSVK** [comment]
tak co je to ok? 

nemal by byt ten headers.cmake skor ako subdirectory?

**Review by kapecp — APPROVED** [review]

**Pull request merged**